### PR TITLE
labor share target in crop production

### PR DESCRIPTION
**Pull request by deleip**
## :bird: Description of this PR :bird:

- This PR allows to set a target labor share (out of labor + capital costs for crop production) to be met by 2050. To allow for regional differences, an adjustment factor (`s38_target_labor_share_adjustment`) is applied to the difference between baseline labor cost share in 2050 and the global target (`s38_target_labor_share`) to define an regionally adjusted target. 

## :wrench: Checklist for PR creator :wrench:

- [x] Label pull request [from the label list](https://github.com/magpiemodel/magpie/labels).
  - **Low risk**: Simple bugfixes (missing files, updated documentation, typos) or changes  in start or output scripts
  - **Medium risk**: Uncritical changes in the model core (e.g. moderate modifications in non-default realizations)
  - **High risk**: Critical changes in model core or default settings (e.g. changing a model default or adjusting a core mechanic in the model)

- [x] Self-review own code
  - No hard coded numbers and cluster/country/region names.
  - The new code doesn't contain declared but unused parameters or variables.
  - [`magpie4`](https://github.com/pik-piam/magpie4) R library has been updated accordingly and backwards compatible where necessary.
  - `scenario_config.csv` has been updated accordingly (important if `default.cfg` has been updated)

- [x] Document changes 
  - Add changes to `CHANGELOG.md`
  - Where relevant, put In-code documentation comments
  - Properly address updates in interfaces in the module documentations
  - run [`goxygen::goxygen()`](https://github.com/pik-piam/goxygen) and verify the modified code is properly documented

- [x] Perform test runs
  - **Low risk**: 
    - Run a compilation check via `Rscript start.R --> "compilation check"`
  - **Medium risk**: 
    - Run test runs via `Rscript start.R --> "test runs"`
    - Check logs for errors/warnings
  - **High risk**:
    - Run test runs via `Rscript start.R --> "test runs"`
    - Check logs for errors/warnings
    - Default run from the PR target branch for comparison
    - Provide relevant comparison plots (land-use, emissions, food prices, land-use intensity,...)

### :chart_with_downwards_trend: Performance changes :chart_with_upwards_trend:
  
  - Current develop branch default : 21 mins
  - This PR's default :  21 mins

## :rotating_light: Checklist for reviewer :rotating_light:

- PR is labeled correctly
- Code changes look reasonable
  - No hard coded numbers and cluster/country/region names.
  - No unnecessary increase in module interfaces
  - model behavior/performance is satisfactory.
- Changes are properly documented
  - `CHANGELOG` is updated correctly
  - Updates in interfaces have been properly addressed in the module documentations
  - In-code documentation looks appropriate
- [ ] content review done (at least 1)
- [x] RSE review done (at least 1)

### CODE REVIEW

File: modules/38_factor_costs/sticky_labor/presolve.gms [hunk 1]
```diff
@@ -52,6 +41,22 @@ else
 i38_ces_shr(j,kcr) = sum(cell(i,j), (p38_intr_depr(t,i) * sum(mobil38, v38_capital_need.l(j,kcr,mobil38))**(1 + s38_ces_elast_par)) / (p38_intr_depr(t,i) * sum(mobil38, v38_capital_need.l(j,kcr,mobil38))**(1 + s38_ces_elast_par) + pm_hourly_costs(t,i,"baseline") * v38_laborhours_need.l(j,kcr)**(1 + s38_ces_elast_par)));
 i38_ces_scale(j,kcr) = sum(cell(i,j), 1/([i38_ces_shr(j,kcr) * sum(mobil38, v38_capital_need.l(j,kcr,mobil38))**(-s38_ces_elast_par) + (1 - i38_ces_shr(j,kcr)) * v38_laborhours_need.l(j,kcr)**(-s38_ces_elast_par)]**(-1/s38_ces_elast_par)));
 
+* minimum labor share based on target and adjustment factor
+if (m_year(t) < s38_fix_capital_need,
```
Comment (bodirsky) [inline modules/38_factor_costs/sticky_labor/presolve.gms:45]: s38_fix_capital need again doesnt sound like the parameter where a minimum labor share kicks in
how about s38_startyear_min_labor?

Comment (bodirsky) [reply modules/38_factor_costs/sticky_labor/presolve.gms:45]: and why is the min year with a parameter, but the target year 2050 is hardcoded? the other way round would seem more logical, or even better both parameters.

File: modules/56_ghg_policy/price_aug22/input.gms [hunk 2]
```diff
@@ -36,7 +36,7 @@ sets
 ;
 
 scalars
-  s56_limit_ch4_n2o_price upper limit for CH4 and N2O GHG prices (USD05MER per tC) / 1000 /
+  s56_limit_ch4_n2o_price upper limit for CH4 and N2O GHG prices (USD05MER per tC) / 4000 /
```
Comment (bodirsky) [inline modules/56_ghg_policy/price_aug22/input.gms:39]: i guess this change is not by you?

**Review by bodirsky — CHANGES_REQUESTED** [review]
naming changes request, content looks fine!

File: modules/38_factor_costs/sticky_labor/presolve.gms (hunk 1 above)
Comment (deleip) [reply modules/38_factor_costs/sticky_labor/presolve.gms:45]: s38_fix_capital was already introduced earlier, to allow comparison between sticky_labor and sticky_feb16. So it's not only about the start year for min_labor scenarios but more general. How about s38_startyear_labor_substitution? But that might be still confusing, as 2020 is not the first year with substitution, but the last year where it's still fixed...

Comment (deleip) [reply modules/38_factor_costs/sticky_labor/presolve.gms:45]: That s38_fix_capital was already there before is also the reason why this is a parameter in the target labor share calculation (for consistency), while 2050 is fixed. I can make the target year a parameter as well...

File: modules/56_ghg_policy/price_aug22/input.gms (hunk 2 above)
Comment (deleip) [reply modules/56_ghg_policy/price_aug22/input.gms:39]: no, Florian changed the default a few weeks back

**Review by bodirsky — APPROVED** [review]

**Review by tscheypidi — APPROVED** [review]

**Pull request merged**